android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1738,8 +1738,10 @@ public void run() {
           }
 
           InputConnection ic = IMService.getCurrentInputConnection();
-          if ((ic == null) && isDebugMode()) {
-            Log.w("SWK: JS Handler", "insertText failed: InputConnection is null");
+          if (ic == null) {
+            if (isDebugMode()) {
+              Log.w("SWK: JS Handler", "insertText failed: InputConnection is null");
+            }
```

Comment:
I assume that this change is intentional (i.e., no longer doing the early return if ic is null)? I don't have enough context or perspective to understand the implications of this. But assuming it's on purpose and makes sense, it LGTM.

Reply:
The crash was occurring in the release builds because it wasn't getting to the `return`.